sushi-config.yaml

```diff
@@ -7,7 +7,7 @@ canonical: http://ehealth.sundhed.dk/fhir
 name: eHealth Infrastructure
 title: eHealth Infrastructure
 status: active # draft | active | retired | unknown
-version: 3.3.0
+version: 3.2.0
```

Comment:
older version?

Reply:
The last version published was 3.1.0
Erik asked me to change this new version to 3.2.0 since naming it 3.3.0 was apparently an accident.